src/components/App.jsx

```diff
@@ -0,0 +1,37 @@
+import 'bootstrap/dist/css/bootstrap.min.css';
+import '../style/App.css'
+import { useState, useEffect } from 'react';
+import { Routes, Route } from 'react-router-dom';
+import { getArticles } from '../utils/api-requests.js';
+import Navigation from './Navigation.jsx';
+import ArticleList from './ArticleList.jsx';
+
+function App() {
+  const [articles, setArticles] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
+
+  useEffect(()=>{
+    setIsLoading(true);
+    getArticles()
+    .then((data)=>{
+      setArticles(data.articles);
+      setIsLoading(false);
+    })
+  },[])
+ 
+
+  return (
+    <>
+      <Navigation />
+      <Routes>
+        <Route path="/" element={<ArticleList isLoading={isLoading} articles={articles} />} />
+        <Route path="/articles" element={<h1>Article test</h1>} />
```

Comment:
With each PR your code should be "production ready" I would hesitate to have placeholder values particularly as you go deeper into the project